src/store/index.js

```diff
@@ -42,7 +47,15 @@ export default new Vuex.Store({
     UPDATE_SORT_OPTION(state, value) {
       state.sortOption = value;
     },
+    POPULATE_MOVIES(state, value) {
+      state.movies = value;
+    },
+  },
+  actions: {
+    LOAD_USERS({ commit }) {
+      return ApiService.getMovies().then((movies) =>
+        commit("POPULATE_MOVIES", movies.data)
```

Comment:
A best practice would be storing action specific names in a "actionTypes" file and using it to access the action name or defining them as dynamic properties.
E.g. `[actionTypes.POPULATE_MOVIES](state, value) {...}`
`commit(actionTypes.POPULATE_MOVIES, movies.data)`
Using this approach you avoid accidental typos in actions across the app.